core/src/main/resources/error/error-classes.json

```diff
@@ -3002,5 +3002,135 @@
     "message" : [
       "Failed to execute command because subquery expressions are not allowed in DEFAULT values."
     ]
+  },
+  "_LEGACY_ERROR_TEMP_2000" : {
+    "message" : [
+      "<message>. If necessary set <ansiConfig> to false to bypass this error."
+    ]
+  },
+  "_LEGACY_ERROR_TEMP_2001" : {
+    "message" : [
+      "<message> If necessary set <ansiConfig> to false to bypass this error"
```

Comment:
Missed a point after `<message>`?

Just in case, _LEGACY_ERROR_TEMP_2001 and _LEGACY_ERROR_TEMP_2000 are the same. Can you merge them, and reuse existing one?

Reply:
I am going to merge this PR since the tests passed. Please, address this minor comment in your next PR https://github.com/apache/spark/pull/38108.

Reply:
Let me address them. Thanks! :-)